src/ja/raw18/src/eu/kanade/tachiyomi/extension/ja/raw18/Raw18.kt

```diff
@@ -0,0 +1,127 @@
+package eu.kanade.tachiyomi.extension.ja.raw18
+
+import eu.kanade.tachiyomi.multisrc.wpcomics.WPComics
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Calendar
+import java.util.Locale
+
+class Raw18 : WPComics("Raw18", "https://raw18.net", "ja", SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'", Locale.JAPANESE), null) {
+    override val searchPath = "search/manga"
+    override val genresUrlDelimiter = "="
+    override fun popularMangaSelector() = "div.items article.item"
+    override fun popularMangaNextPageSelector() = "li:nth-last-child(2) a.page-link"
+    override fun searchMangaSelector() = popularMangaSelector()
+    override fun mangaDetailsParse(document: Document): SManga {
+        return SManga.create().apply {
+            document.select("article#item-detail").let { info ->
+                author = info.select("li.author p.col-xs-8").text()
+                status = when {
+                    info.select("li.status p.col-xs-8").text().contains("Ongoing", true) -> SManga.ONGOING
+                    else -> SManga.UNKNOWN
+                }
+                genre = info.select("li.kind p.col-xs-8 a").joinToString { it.text() }
+                description = info.select("div.detail-content").text()
+                thumbnail_url = imageOrNull(info[0].selectFirst("div.col-image img")!!)
+            }
+        }
+    }
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val filterList = filters.let { if (it.isEmpty()) getFilterList() else it }
+        val url = "$baseUrl/search/manga".toHttpUrl().newBuilder()
+        filterList.forEach { filter ->
+            when (filter) {
+                is GenreFilter -> filter.toUriPart()?.let { url.addQueryParameter("genre", it) }
+                is StatusFilter -> filter.toUriPart()?.let { url.addQueryParameter("status", it) }
+                else -> {}
+            }
+        }
+        url.apply {
+            addQueryParameter(queryParam, query)
+            addQueryParameter("page", page.toString())
+        }
+        return GET(url.build(), headers)
+    }
+    override fun chapterFromElement(element: Element): SChapter {
+        val minuteWords = listOf("minute", "分")
+        val hourWords = listOf("hour", "時間")
+        val dayWords = listOf("day", "日")
+        val weekWords = listOf("week", "週間")
+        val monthWords = listOf("month", "月")
+        val chapterDate = element.select("div.col-xs-4").text()
+        return SChapter.create().apply {
+            element.select("a").let {
+                name = it.text()
+                setUrlWithoutDomain(it.attr("href"))
```

Comment:
```suggestion
                setUrlWithoutDomain(it.absUrl("href"))
```

`it` must be of type `Element`, not `Elements`. You can use `document.selectFirst()` for that